public/pages/MonitorDetails/components/MonitorOverview/utils/getOverviewStats.js

```diff
@@ -49,12 +53,25 @@ function getMonitorType(searchType) {
   }
 }
 
+function getMonitorLevelType(monitorType) {
+  console.log(monitorType);
```

Comment:
Can remove the `console.log` if it's not needed

Reply:
Right, let me remove that.

Reply:
Removed